src/pages/Calendar/Calendar-FlowFilter.vue

```diff
@@ -94,8 +93,14 @@ export default {
             accum[run.flows[0].id] = {
               name: run.flows[0].name,
               active: false,
-              id: run.flows[0].id
+              id: run.flows[0].id,
+              version: run.flows[0].version,
+              fgId: run.id
             }
+          } else {
+            accum[run.flows[0].id].name = run.flows[0].name
+            accum[run.flows[0].id].version = run.flows[0].version
+            accum[run.flows[0].id].fgId = run.id
```

Comment:
It might be cleaner to store the flow or flow id in a local variable somewhere in this enclosure instead, and add some optional chaining/checks to make sure it's defined.